schema.json

```diff
@@ -489,426 +489,438 @@
-        "options": [
-          {
-            "value": "black",
-            "label": "Black (white text)"
-          },
-          {
-            "value": "white",
-            "label": "White (black text)"
-          },
-          {
-            "value": "white-border",
-            "label": "White (black text with border)"
-          }
-        ]
-      },
-      {
-        "type": "heading",
-        "content": "GeoTrust SSL"
-      },
-      {
-        "type": "paragraph",
-        "content": "If you've purchased a GeoTrust SSL from BigCommerce, check your BigCommerce Account Dashboard for the correct Common Name to use here."
-      },
-      {
-        "type": "text",
-        "label": "SSL Common Name",
-        "force_reload": true,
-        "id": "geotrust_ssl_common_name"
-      },
-      {
-        "type": "select",
-        "label": "Seal size",
-        "id": "geotrust_ssl_seal_size",
-        "options": [
-          {
-            "value": "M",
-            "label": "Medium"
-          },
-          {
-            "value": "S",
-            "label": "Small"
-          }
-        ],
-        "force_reload": true
-      }
+        {
+            "type": "heading",
+            "content": "Header"
+        },
+        {
+            "type": "color",
+            "label": "Header background",
+            "id": "header-backgroundColor"
+        },
+        {
+            "type": "heading",
+            "content": "Logo"
+        },
+        {
+            "type": "select",
+            "label": "Logo position",
+            "id": "logo-position",
+            "force_reload": true,
+            "options": [
+                {
+                    "value": "right",
+                    "label": "Right"
+                },
+                {
+                    "value": "center",
+                    "label": "Center"
+                },
+                {
+                    "value": "left",
+                    "label": "Left"
+                }
+            ]
+        },
+        {
+            "type": "imageDimension",
+            "label": "Logo image size",
+            "id": "logo_size",
+            "description": "Please specify a maximum image size for desktop display. We'll automatically scale down the image for smaller devices.",
+            "force_reload": true,
+            "options": [
+                {
+                    "value": "250x100",
+                    "label": "Optimized for theme"
+                },
+                {
+                    "value": "original",
+                    "label": "Original (as uploaded)"
+                },
+                {
+                    "value": "custom",
+                    "label": "Specify dimensions"
+                }
+            ]
+        },
+        {
+            "type": "select",
+            "label": "Logo font size",
+            "id": "logo_fontSize",
+            "options": [
+                {
+                    "value": 14,
```

Comment:
Same thing here - we should limit this changeset to just the new feature.

Reply:
Updated the schema.json file with an editor that doesn't reformat the spacing and tabs in the file.